test/files.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+
+module.exports = (common) => {
+  describe('.file', () => {
+    let ipfs
+
+    before((done) => {
+      common.setup((err, _ipfs) => {
+        expect(err).to.not.exist
+        ipfs = _ipfs
+        done()
+      })
+    })
+
+    after((done) => {
+      common.teardown(done)
+    })
+
+    describe('.cat', () => {
+      it('returns file stream', (done) => {
+        const hash = 'QmT78zSuBmuS4z925WZfrqQ1qHaJ56DQaTfyMUF7F8ff5o'
+        ipfs.files.cat(hash, (err, res) => {
+          expect(err).to.not.exist
+          res.on('data', (data) => {
+            data.stream.pipe(bl((err, bldata) => {
+              expect(err).to.not.exist
+              expect(bldata.toString()).to.equal('hello world\n')
+              done()
+            }))
+          })
+        })
+      })
+    })
+  })
+}
```

Comment:
It would be good to also have a test for a non existing hash, so that we can also test for the err behaviour